### DISCUSSION TITLE
Search value on object painel

### DISCUSSION THREAD

**Discussion by Momoshikin** [zero-plusplus/vscode-autohotkey-debug#174] [Q&A] [answered]
When you hover an object and it pops this window:
![image](https://user-images.githubusercontent.com/92819425/144296940-7d4a412e-9d26-4e51-a87c-289aa3734ed4.png)

Is possible to search for a specific value? or an object key?

**Comment by zero-plusplus** [comment, marked as answer]
As far as I know, it is not possible to search by hover information.

The variable view of the data inspection allows incremental search by variable name or object key name.

To use this, simply focus on the variable view or the object variable you want to examine and enter the string you want to search for. 

By the way, this incremental search can be used anywhere in the treeview format, for example in Explorer.

---

I think vscode probably does not support searching for variable value.

If such a UI is supported, I will implement it eventually.
However, I will delay the implementation for about a year after vscode supports it.

This is because I need to raise the required vscode version of the extension to implement it.
The latest version of vscode may contain bugs, so I need to give it some time.
